src/datasources/cache/cache.router.ts

```diff
@@ -51,7 +51,6 @@ export class CacheRouter {
     );
   }
 
-  // TODO: remove this prefixed key if eventually only one balances provider is used
```

Comment:
This is not applicable anymore, since it will be necessary to have prefixes as Transaction Service will be used as a fallback, and a distinction between providers still will be needed.